quadlet/redpanda/redpanda.container

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=A RedPanda single node development cluster
+After=network-online.target
+
+[Container]
+Image=docker.redpanda.com/redpandadata/redpanda:latest
+ContainerName=redpanda-server
+Exec=redpanda start --overprovisioned --smp 1 --kafka-addr internal://redpanda-server:9092 --advertise-kafka-addr internal://redpanda-server:9092
```

Comment:
Can this exec be split onto multiple lines somehow?  Like you can with a docker-compose

The Redpanda docs show an extremely long `command` string, which would be difficult to maintain if someone needed something similar ([here](https://docs.redpanda.com/current/reference/docker-compose/)).

Reply:
Not sure why the comment isn't linked to this conversation
https://github.com/containers/appstore/pull/14#discussion_r1436064443

Yes, I believe it can be.  Will try switching these flags over to a config file.